test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/PatchTests.cs

```diff
@@ -51,7 +51,6 @@ public async Task GivenAServerThatSupportsIt_WhenSubmittingALevel1PropertyPatch_
         [InlineData("[{\"op\":\"replace\",\"value\":\"female\"}, {\"op\":\"remove\",\"path\":\"/address\"}]")]
         [InlineData("[{\"op\":\"coo\",\"path\":\"/gender\",\"value\":\"female\"}, {\"op\":\"remove\",\"path\":\"/address\"}]")]
         [InlineData("[{\"op\":\"replace\",\"path\":\"\",\"value\":\"\"}]")]
-        [InlineData("[{\"op\":\"replace\",\"path\":\"/gender\"}]")]
```

Comment:
>[InlineData("[{\"op\":\"replace\",\"path\":\"/gender\"}]")] [](http://example.com/codeflow?start=8&length=59)

This used to throw error, but with aspnet 6.0 changes it's totally fine to omit value, it would be treated as null and set gender to become empty value.